Sources/Router/router.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import Requests
+import Responses
+
+public class Router {
+
+    private var routes: [String: [String]]
+    private var errorMessage: String = "<p> URL does not exist </p>"
+
+    private func routeSetUp() {
+        routes["/"] = ["GET", "POST", "PUT"]
+        routes["/form"] = ["GET", "POST", "PUT"]
```

Comment:
You might want to use enums here for GET, POST, etc. Swift enums have some nice properties